chains.yaml

```diff
@@ -90,6 +90,42 @@
   build-env:
     - BUILD_TAGS=muslc
 
+# Carbon
+- name: carbon
+  github-organization: Switcheo
+  github-repo: carbon-bootstrap
+  language: rust
+  build-target:
+  pre-build: |
+    apt update && apt install wget build-essential jq cmake sudo -y
+    wget https://github.com/google/leveldb/archive/1.23.tar.gz && \
+      tar -zxvf 1.23.tar.gz && \
+      wget https://github.com/google/googletest/archive/release-1.11.0.tar.gz && \
+      tar -zxvf release-1.11.0.tar.gz && \
+      mv googletest-release-1.11.0/* leveldb-1.23/third_party/googletest && \
+      wget https://github.com/google/benchmark/archive/v1.5.5.tar.gz && \
+      tar -zxvf v1.5.5.tar.gz && \
+      mv benchmark-1.5.5/* leveldb-1.23/third_party/benchmark && \
+      cd leveldb-1.23 && \
+      mkdir -p build && \
+      cd build && \
+      cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON .. && \
+      cmake --build . && \
+      sudo cp -P libleveldb.so* /usr/local/lib/ && \
+      sudo ldconfig && \
+      cd .. && \
+      sudo cp -r include/leveldb /usr/local/include/ && \
+      cd .. && \
+    VERSION=2.0.0
+    NETWORK=mainnet
+    wget https://github.com/Switcheo/carbon-bootstrap/releases/download/v${VERSION}/carbond${VERSION}-${NETWORK}.linux-$(dpkg --print-architecture).tar.gz
```

Comment:
If we use the `VERSION` var that comes from the Dockerfile, which will have the `v`, this can be
`wget https://github.com/Switcheo/carbon-bootstrap/releases/download/${VERSION}/carbond${VERSION:1}-${NETWORK}.linux-$(dpkg --print-architecture).tar.gz`

Reply:
When I try this, I'm getting a `Bad substitution` Error. It can't seem to remove that first character.

Reply:
It's hacky... but instead of using the colon, this works:
`VERSION_NUM=$(echo ${VERSION} | cut -c 2-)`